ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -100,46 +108,63 @@ public struct Avatar: View, TokenizedControlView {
 
     public var body: some View {
         let style = state.style
+        let size = state.size
+
+        let initialsString: String = ((style == .overflow) ? state.primaryText ?? "" : Avatar.initialsText(fromPrimaryText: state.primaryText,
+                                                                                                           secondaryText: state.secondaryText))
+        let shouldUseCalculatedColors = !initialsString.isEmpty && style != .overflow
+        let shouldUseDefaultImage = (state.image == nil && initialsString.isEmpty && style != .overflow)
+
+        let activityStyle = state.activityStyle
         let presence = state.presence
-        let shouldDisplayPresence = presence != .none
+        let shouldDisplayActivity = state.activityImage != nil && activityStyle != .none && (style == .default && !shouldUseDefaultImage)
```

Comment:
I think this is looking for the wrong image. I think we should be checking for if `activityImage != nil`, rather than `!shouldUseDefaultImage`

Reply:
No this is correct. Design only wants the activity image to be shown in default style when it's not using default image. I'm already checking in the beginning of this line if the activityImage is nil